xmtp_mls/src/storage/encrypted_store/consent_record.rs

```diff
@@ -85,8 +85,8 @@ impl DbConnection {
 #[diesel(sql_type = Integer)]
 /// Type of consent record stored
 pub enum ConsentType {
-    /// Consent is for a group
-    GroupId = 1,
+    /// Consent is for a conversation
+    ConversationId = 1,
```

Comment:
Since it is just a number under the hood this rename should have no effect on existing records.